plugins/repoclosure.py

```diff
@@ -75,6 +75,19 @@ def _get_unresolved(self, arch=None):
         if arch is not None:
             available = available.filter(arch=arch)
         pkgs = set()
+
+        # available seems to contain latest-per-repo but we want the latest
```

Comment:
My use case is that I have a local repository that pulls in dependencies from Fedora or CentOS+EPEL. I want to check that all packages in my local repo have resolvable dependencies but I don't want the output cluttered up with details of unresolvable dependencies from Fedora/CentOS/EPEL, and I don't want to spend any time (which would be quite significant) checking those dependencies either. So this approach gives me teh data I need, and much more quickly than doing a full repoclosure over all of the repos involved. This approach works for me but I'd certainly be open to other approaches.